js/iframe-coordinator/Elm.d.ts

```diff
@@ -1,69 +1,58 @@
-declare module "*/Client.elm" {
```

Comment:
This looks like a lot of changes, but mostly it's me discovering that you don't have to explicitly export types that are used in other exports, and renaming `Worker` and `HostRouter` to `ClientProgram` and `HostProgram` to better match the original source of those types.